pkg/config/setup/config.go

```diff
@@ -1069,6 +1069,12 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.image_tag", "latest")
 	config.BindEnv("admission_controller.cws_instrumentation.init_resources.cpu")
 	config.BindEnv("admission_controller.cws_instrumentation.init_resources.memory")
+	config.BindEnvAndSetDefault("admission_controller.agent_sidecar.enabled", false)
+	config.BindEnvAndSetDefault("admission_controller.agent_sidecar.endpoint", "/agentsidecar")
+	// Should be able to parse it to []map[string]string
+	config.BindEnvAndSetDefault("admission_controller.agent_sidecar.label_selectors", "[]")
+	// Should be able to parse it to []map[string]string
+	config.BindEnvAndSetDefault("admission_controller.agent_sidecar.namespace_selectors", "[]")
```

Comment:
I thought we said that for the first implementation we only support `labels_selectors` with the default label key `agent.datadoghq.com/sidecar`

Reply:
I remember this was an initial thought, but after that based on the following discussions, I understood that we use namespace and label selectors and constrain user to specify only one selector.

With this, if users want to use `agent.datadoghq.com/sidecar` , they could, but they also have the flexibility to configure selectors based on namespaces (or multiple labels) for example.

Reply:
With the current scope we support generic object and namespace selectors [AC selectors](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-objectselector) to target pods (only). This way users can configure targeting in AC instead of configuring their applications.